style.scss

```diff
@@ -0,0 +1,532 @@
+    --size-400: 1rem; // 16px
+    --size-500: 1.5rem; // 24px
+    --size-600: 2rem; // 32px
+    --size-700: 3rem; // 48px
+    --size-800: 4rem; // 64px
+    --size-900: 5rem; // 80px
+}
+
+/*
+  1. Use a more-intuitive box-sizing model.
+  */
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+/*
+2. Remove default margin
+*/
+* {
+    margin: 0;
+}
+
+/*
+3. Allow percentage-based heights in the application
+*/
+html,
+body {
+    height: 100%;
+}
+
+/*
+6. Improve media defaults
+*/
+img,
+picture,
+video,
+canvas,
+svg {
+    display: block;
+    max-width: 100%;
+}
+
+/*
+7. Remove built-in form typography styles
+*/
+input,
+button,
+textarea,
+select {
+    font: inherit;
+}
+
+/*
+8. Avoid text overflows
+*/
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    overflow-wrap: break-word;
+}
+
+
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
+ul[role='list'],
+ol[role='list'] {
+    list-style: none;
+    padding: 0;
+}
+
+/*
+  Typographic tweaks!
+  4. Add accessible line-height
+  5. Improve text rendering
+*/
+
+
+.flex {
+    display: flex;
+
+    &-row {
+        flex-direction: row;
+    }
+    
+    &-col {
+        flex-direction: column;
+    }
+
+    &.justify-between {
+        justify-content: space-between;
+    }
+    
+    &.items-center {
+        align-items: center;
+    }
+    
+    &.gap-4{
+        gap: var(--size-400);
+    }
+}
+
+.text {
+    &-center {
+        text-align: center;
+    }
+    
+    &-right {
+        text-align: right;
+    }
+    
+    &-left {
+        text-align: left;
+    }
+}
+
+.button {
+    cursor: pointer;
+    font-weight: var(--fw-bold);
+    font-size: var(--fs-button);
+    line-height: 1;
+    color: var(--clr-neutral-100);
+    background-color: var(--clr-primary-900);
+    text-decoration: none;
+    border: 0;
+    border-radius: 100vmax;
+    padding: var(--size-400) var(--size-500);
+    max-height: 3.5rem;
+
+    &:hover,
+    &:focus-visible {
+        background-color: var(--clr-primary-900-hover);
+    }
+
+    &[data-type='alternative'] {
+        background-color: var(--clr-accent-400);
+
+        &:hover,
+        &:focus-visible {
+            background-color: var(--clr-accent-400-hover);
+        }
+    }
+    
+    &--previous,
+    &--next{
+        border-radius: 50%;
+    }
+}
+
+.card {
+    border-radius: var(--size-200);
+}
+
+/* general styling */
+body {
+    -webkit-font-smoothing: antialiased;
+    font-family: var(--ff-body);
+    font-size: var(--fs-body);
+    font-weight: var(--fw-regular);
+    color: var(--clr-primary-900);
+}
+
+.container {
+    --max-width: 1110px;
+    --padding: 1rem;
+
+    width: min(var(--max-width), 100% - (var(--padding) * 2));
+    margin-inline: auto;
+}
+
+.header {
+    padding: var(--size-400) 0;
+    &__logo {
+        // display: flex;
+        // align-items: center;
+        // gap: 1rem;
+        // padding: 1rem 0;
+    }
+}
+
+.portfolio {
```

Comment:
If you are not using portfolio class for any styling, maybe it would be better not to use it as a block and avoid unnecessary nesting?

Reply:
I meant not to use it as a separate block at all.
In HTML instead of:

container portfolio
    protfolio__intro
    protfolio__skills
It could be just:

container
   portfolio-intro
   portfolio-skills
Where they are as blocks. Then portfolio-skills__item, portfolio-skills__text etc.
Because now you still have .portfolio__skills { } where you only nest other classes and don't have style or anything.
But it's ok, as it is now. You can leave the portfolio block.